plugins/jvm-abi-gen/src/org/jetbrains/kotlin/jvm/abi/JvmAbiOutputExtension.kt

```diff
@@ -69,13 +74,15 @@ class JvmAbiOutputExtension(
                     // Note that outputFile may be null, e.g., for empty $DefaultImpls classes in the JVM backend.
                     outputFile == null ->
                         null
```

Comment:
Let's move it one line above into `if (outputFile == null) return@mapNotNull null`, and make the `when` exhaustive via `when (abiInfo) ...`